int32 INFO = 0
int32 WARN = 1
int32 ERROR = 2

string message
int32 level
uint32 timeout
